src/app/shared/services/fill-filter.service.ts

```diff
@@ -0,0 +1,62 @@
+import { HttpClient, HttpHeaders } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+import { from, Observable } from "rxjs";
+import {client_id, api_token} from "../../../assets/creds"
+
+@Injectable({
+    providedIn:'root'
+})
+export class FillFilterService {
+
+  constructor(private http: HttpClient){   }
+
+  getRatings(): Observable<[{id: number, rating: number}]> {
```

Comment:
As discussed, interfaces are needed here, create a separate folder (or several) for them